src/PostgREST/MediaType.hs

```diff
@@ -106,26 +112,44 @@ toMimePlanFormat PlanText = "text"
 --
 -- >>> decodeMediaType "application/vnd.pgrst.plan+json;for=\"text/csv\""
 -- MTPlan MTTextCSV PlanJSON []
+--
+-- >>> decodeMediaType "application/json;nulls=stripped"
+-- MTArrayJSONStrip
```

Comment:
As mentioned on https://github.com/PostgREST/postgrest/issues/1601#issuecomment-703323701, we cannot add parameters to existing media types (`application/json`). At least not without an RFC. 

So `application/json` should not accept nulls=stripped.